snovault/tests/test_schema_utils.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 from snovault.schema_utils import (
-    _update_resolved_data, _handle_list_or_string_value, resolve_merge_refs
+    _update_resolved_data, _handle_list_or_string_value, resolve_merge_refs,
+    validate
```

Comment:
I predict PyCharm and perhaps `make lint` will fuss about you importing things with leading `_`, so I would write:
```suggestion
    _update_resolved_data, _handle_list_or_string_value,  # noQA - testing protected members
    resolve_merge_refs, validate
```